services/QuillLMS/spec/controllers/concerns/slack_tasks_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe SlackTasks do
+  include SlackTasks
+
+  let(:sales_form_submission) { create(:sales_form_submission) }
+
+  before do
+
+    allow(ENV).to receive(:[]).with('RAILS_ENV').and_return('production')
+    ENV['SLACK_API_WEBHOOK_SALES'] = 'slack-test.com'
```

Comment:
You'll need to stub the ENV constants rather than set them directly.  The reason for this is if you don't, those settings will carry over to other tests.  

```
    before { stub_const('ENV', {'RAILS_ENV' => 'production', 'SLACK_API_WEBHOOK_SALES' => 'slack-test.com'}) }
```

Reply:
For example: 
![Screen Shot 2022-05-12 at 9 49 25 AM](https://user-images.githubusercontent.com/2057805/168090552-e4bbd489-a72b-4b39-a89a-0c999b17491e.png)